config/services.yaml

```diff
@@ -60,7 +60,11 @@ parameters:
     html5_validation: true
 
     front_sort_options: top|hot|active|newest|oldest|commented
```

Comment:
`commented` isn't valid for post/entry single so maybe worth breaking out to a new var